VideoApp/VideoApp/Assets.xcassets/CallKitIcon.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```

Comment:
These are just placeholder images from the Apple sample app until our UX designer provides new ones.